documentation/src/main/asciidoc/querylanguage/Relational.adoc

```diff
@@ -110,8 +110,8 @@ select book.isbn,
 from Item
 where book.isbn = :isbn
 group by year(order.dateTime)
-having year(order.dateTime) > 2003
-   and sum(quantity) > 0
+having year > 2003
+   and yearlyTotalSold > 0
```

Comment:
not 100% sure, but seems usually having clause taps into aliases in the projection clause without duplication

Reply:
Group by and order by clauses support using aliases, but other clauses don't.

Reply:
I just ran this code on postgres and it worked:

```java
			sessionFactory.inSession(s -> s.createSelectionQuery(
					"select sum(book.quantity), year(book.publicationDate) from Book book " +
					"group by year(book.publicationDate) " +
					"having year(book.publicationDate) > 2003 " +
					"   and sum(book.quantity) > 0").getResultList());
```

Reply:
I reverted it back BEFORE I saw your comment. I googled further (e.g. https://stackoverflow.com/questions/72314037/how-can-i-use-an-alias-with-having-clause), so it seems the feature might not be supported by all DBs?
For that reason, I refrained from reverting back what I reverted.